app/pb/api.proto

```diff
@@ -55,6 +55,12 @@ service API {
   rpc InitiateMediaUpload(google.protobuf.Empty) returns (InitiateMediaUploadRes) {
     // Initiate a media upload
   }
+
+  rpc BlockUser(UserBlockingReq) returns (google.protobuf.Empty);
+
+  rpc GetBlockedUsers(google.protobuf.Empty) returns (GetBlockedUsersRes);
+
+  rpc UnblockUser(UserBlockingReq) returns(google.protobuf.Empty);
```

Comment:
Please adhere to the style of `rpc SomeFunction` takes request `SomeFunctionReq`, so `BlockUser` should take `BlockUserReq`. Please also comment these functions!

It might also be good to separate these out into a separate `blocking.proto`, or similar?